backend/base/views.py

```diff
@@ -1,17 +1,32 @@
 from django.shortcuts import render
+from geopy import distance
+from pyexpat import model
 from rest_framework import status, viewsets
 from rest_framework.decorators import action, api_view
 from rest_framework.response import Response
 
 # Create your views here.
 from . import models, serializers
 
+taking_attendence = False
+FOODYS = (12.968977, 79.158283)
```

Comment:
Maybe let's not hardcode the values here, you can keep them as env variables as a JSON, loaded on runtime and parsed.

Reply:
yeah I will be putting these into the State Model
so it can be different for every club

Reply:
sure, that works.